server/src/main/java/com/linecorp/centraldogma/server/command/Command.java

```diff
@@ -48,6 +48,9 @@
         @Type(value = RemoveRepositoryCommand.class, name = "REMOVE_REPOSITORY"),
         @Type(value = PurgeRepositoryCommand.class, name = "PURGE_REPOSITORY"),
         @Type(value = UnremoveRepositoryCommand.class, name = "UNREMOVE_REPOSITORY"),
+        @Type(value = ApplyingDiffPushCommand.class, name = "APPLYING_DIFF_PUSH"),
+        @Type(value = ReplicationPushCommand.class, name = "REPLICATION_PUSH"),
+        // Deprecated. Should use OriginalPushCommand or ReplicationPushCommand.
         @Type(value = PushCommand.class, name = "PUSH"),
```

Comment:
How about this:

- Rename `PushCommand` to `PushAsIsCommand` or `JustPushCommand`.
- Remove `ReplicationPushCommand` and use `PushAsIsCommand`.
- Rename `ApplyingDiffPushCommand` to `NormalizingPushCommand`.

Reply:
Those are good suggestions. Applied. 😄